.github/workflows/test.yml

```diff
@@ -5,41 +5,47 @@ on:
       - master
       - "[0-9]+.[0-9]+.x"
   pull_request:
-    types: [opened, synchronize]
 
 jobs:
   test:
+    name: Run tests on Node.js ${{ matrix.node-version }}
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        node_version: [14, 16, 18, 20, 22]
+        node-version: [14, 16, 18, 20, 22]
```

Comment:
Why the var name change?

Reply:
Convention, all GitHub Actions use this type of casing.